packages/server/__tests__/api/keywords.test.js

```diff
@@ -34,10 +34,43 @@ describe('`/api/keywords` endpoint', async () => {
         },
     };
 
+    let testKeywordsByAgency = null;
+
     before(async function beforeHook() {
         this.timeout(9000); // Getting session cookies can exceed default timeout.
-        fetchOptions.admin.headers.cookie = await getSessionCookie('michael@nv.gov');
+        fetchOptions.admin.headers.cookie = await getSessionCookie('admin1@nv.gov');
         fetchOptions.staff.headers.cookie = await getSessionCookie('user1@nv.gov');
+
+        // Tests below assume the presence of at least one keyword per agency. Previously, we had
+        // a db seed that created some default COVID-related keywords but now that that is gone, we
+        // create some here to avoid dependence on the global keywords seed.
+        // Note: this does not dedupe agency IDs and that's by design; some of the tests need to
+        // delete multiple keywords from the same agency.
+        const allAgencyIds = Object.values(agencies.admin).concat(Object.values(agencies.staff));
+        const testKeywords = allAgencyIds.map((agency_id) => ({
+            mode: 'autoinsert ALL keywords matches',
+            search_term: 'test_keyword',
+            notes: '',
+            agency_id,
+        }));
+        const createdKeywords = await knex(TABLES.keywords)
+            .insert(testKeywords)
+            .onConflict('id')
+            .merge()
+            .returning(['id', 'agency_id']);
+        testKeywordsByAgency = _.groupBy(createdKeywords, 'agency_id');
+    });
+
+    after(async () => {
+        // Delete keywords created by this test to avoid impacting other tests
```

Comment:
hmm in the future we might want to look at the seeding and maybe clear and seed the db before every test

Reply:
I like that conceptually, I'm not sure how much it would slow down running the tests overall. If it does have an impact, it could be annoying for local development since right now there doesn't seem to be a good way to run a single test/suite, we just run everything every time. I'm not going to try to do this now but I agree it's worth looking at in the future.